gen/DocumentFormat.OpenXml.Generator.Models/Models/SchemaType.cs

```diff
@@ -11,6 +11,8 @@ public class SchemaType
 
     public bool IsPart => !string.IsNullOrEmpty(Part);
 
+    public bool IsRootElement => IsPart || BaseClass == "OpenXmlPartRootElement" || BaseClass == "OpenXmlPartRootElement";
```

Comment:
@twsouthwick it looks like one of these `BaseClass == "OpenXmlPartRootElement"` conditions might be removed 😃